docs/setup/manual-setup.md

```diff
@@ -3,41 +3,31 @@
 ## Requirements on your server
 
 - Node.js 10.13 or up
-
-- Database (PostgreSQL, MySQL, MariaDB, SQLite, MSSQL). Must use charset `utf8`: this is typically the
-  default in PostgreSQL and SQLite, while in MySQL and MariaDB UTF-8 might need to be set with
-  `alter database <DBNAME> character set utf8 collate utf8_bin;`
+- Database (PostgreSQL, MySQL, MariaDB, SQLite, MSSQL)  
+  The database must use charset `utf8`. This is typically the default in PostgreSQL and SQLite.  
+  In MySQL and MariaDB UTF-8 might need to be set with `alter database <DBNAME> character set utf8 collate utf8_bin;`  
   Be aware of older MySQL and MariaDB versions which sometimes use shorter representations of UTF-8 than 4 bytes.
   This can break if symbols with more bytes are used.
   You can use `alter database <DBNAME> character set utf8mb4 COLLATE utf8mb4_unicode_ci` to be on the safe side.
-
-- npm (and its dependencies, [node-gyp](https://github.com/nodejs/node-gyp#installation))
-
-- yarn
-
+- Npm (and its dependencies, [node-gyp](https://github.com/nodejs/node-gyp#installation))
+- Yarn
 - Bash (for the setup script)
-
-- For **building** HedgeDoc we recommend to use a machine with at least **2GB** RAM
+- For **building** the HedgeDoc frontend you need a machine with at least **2 GB** RAM.
+    - Starting with release 1.7 the release tarball includes the frontend, so building it yourself is not necessary.
 
 ## Instructions
 
 1. Check if you meet the [requirements at the top of this document](#requirements-on-your-server).
-
-2. Clone this repository (preferred) or download a release and unzip it.
-
-3. Enter the directory and type `bin/setup`, which will install npm dependencies and create configs.
-
+2. Download a [release](https://github.com/hedgedoc/hedgedoc/releases) tarball and extract it.
```

Comment:
I would add an explicit node that one can either clone the repository using the `-b` parameter to get the stable version or use a release.

Reply:
Fixed in the following line